src/fetch/contracts.py

```diff
@@ -107,6 +110,40 @@ def contracts(self, load_from: NetworkFile) -> dict[str, bool]:
         write_to_csv(data_list=confirmed_contracts, outfile=load_file)
         return results
 
+    def get_non_wallets(self, contracts: list[str]):
+        code_at = self.batch_call(
+            addresses=contracts,
+            func=self._get_code_at
+        )
+        collector = defaultdict(list)
+        for account, code in code_at.items():
+            collector[code].append(account)
+
+        total_wallets = 0
+        wallets = set()
+        not_wallets = {}
+        unverified = {}
```

Comment:
It doesn't look like the mapping of code to accounts in `unverified` gets used (i.e. look up) at any point.
In that case `unverified` could probably be a set which would simplify the important parts of the loop to this:
```python
            if byte_code in WALLET_BYTECODE:
                wallets |= set(accounts)
            elif byte_code in UNVERIFIED_BYTECODE:
                unverified |= set(accounts)
            # ...
                
            total_wallets = len(wallets | unverified)
```

Reply:
Right. It used to be used but this has been removed. Thanks.